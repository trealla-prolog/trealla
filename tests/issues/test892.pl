:- initialization(main).

main :-
	E =.. "!!", E.
