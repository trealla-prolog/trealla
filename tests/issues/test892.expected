   Warning: Initialization goal exception: error(existence_error(procedure,! / 1),! / 1).
